fre/tests/test_fre_cmor_cli.py

```diff
@@ -68,7 +85,7 @@ def test_cli_fre_cmor_run_case1(capfd):
         f"{full_outputdir}/sos_Omon_PCMDI-test-1-0_piControl-withism_r3i1p1f1_gn_199307-199807.nc"
 
     # FYI
-    filename = 'ocean_monthly_1x1deg.199301-199712.sos.nc' # unneeded, this is mostly for reference
+    filename = 'reduced_ocean_monthly_1x1deg.199301-199712.sos.nc' # unneeded, this is mostly for reference
```

Comment:
@ceblanton  seems like the down-stream test uses the generated file and succeeds in pipeline!

Reply:
https://github.com/NOAA-GFDL/fre-cli/pull/215/files#r1811014469

Reply:
The downstream test cmorizes the file? I'm still wondering how you created any data for the netcdf file. without data it's not a fair test!

Reply:
i never got around to replying here, but @ceblanton was correct back then and this is now the case in @Ciheim 's `.cdl` file